system/security/plugins/jwtSecurity/jwtSecurity.cpp

```diff
@@ -113,21 +113,20 @@ class CJwtSecurityManager : implements IDaliLdapConnection, public CBaseSecurity
         }
 
         if (clientID.empty())
-            throw MakeStringException(-1, "CJwtSecurityManager: clientID not found in configuration");
+            makeStringException(-1, "CJwtSecurityManager: clientID not found in configuration");
```

Comment:
needs to be

   **throw** makeStringException()

Reply:
<embarrassed/>

Fixed, here and elsewhere.

Reply:
You were checking the code review was thorough...